sormas-e2e-tests/src/test/java/org/sormas/e2etests/helpers/WebDriverHelpers.java

```diff
@@ -389,6 +390,20 @@ public void scrollToElement(final Object selector) {
     waitForPageLoaded();
   }
 
+  public void hoverToElement(By selector) {
+    WebElement menuOption = baseSteps.getDriver().findElement(selector);
+    Actions actions = new Actions(baseSteps.getDriver());
+    try {
+      assertHelpers.assertWithPoll20Second(
+          () -> {
+            scrollToElement(selector);
+            actions.moveToElement(menuOption).perform();
+            waitUntilIdentifiedElementIsVisibleAndClickable(selector);
+          });
+    } catch (Exception ignored) {
```

Comment:
log a message here to be able to debug the issue in logs and understand what happens during execution :D

Reply:
done